gnucash/gnome/gnc-plugin-basic-commands.c

```diff
@@ -113,7 +113,7 @@ static GtkActionEntry gnc_plugin_actions [] =
         G_CALLBACK (gnc_main_window_cmd_file_new)
     },
     {
-        "FileOpenAction", "document-open", N_("_Open..."), "<primary>o",
+        "FileOpenAction", "document-open", N_("_Open…"), "<primary>o",
```

Comment:
About the usage of ellipsis see the discussion in https://bugs.gnucash.org/show_bug.cgi?id=792975